File: editorial/chap3-4/prob3-4-4.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		// 入力
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		
		// 答えの計算
		double Answer = 0.0;
		for (int i = N; i >= 1; i--) {
			Answer += 1.0 * N / i;
		}
		
		// 出力
		System.out.format("%.12f\n", Answer);
	}
}

File: editorial/chap3-4/prob3-4-4.c
#include <stdio.h>

int N;
double Answer = 0;

int main() {
	// 入力
	scanf("%d", &N);
	
	// 期待値を求める
	for (int i = N; i >= 1; i--) {
		Answer += 1.0 * N / i;
	}
	
	// 出力
	printf("%.12lf\n", Answer);
	return 0;
}